src/main/java/appeng/core/Registration.java

```diff
@@ -107,6 +107,7 @@
 import appeng.recipes.handlers.Shapeless;
 import appeng.recipes.handlers.Smelt;
 import appeng.recipes.ores.OreDictionaryHandler;
+import appeng.recipes.RecipeHandler;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Wrong order for 'appeng.recipes.RecipeHandler' import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)